packages/react-meteor-data/createContainer.jsx

```diff
@@ -2,6 +2,7 @@
  * Container helper using react-meteor-data.
  */
 
+import createClass from 'create-react-class';
```

Comment:
To make sure we stay in-line with the [React docs](https://facebook.github.io/react/docs/react-without-es6.html), can we change `createClass` to `createReactClass`?